null_safety_examples/misc/lib/samples/spacecraft.dart

```diff
@@ -0,0 +1,87 @@
+// ignore_for_file: sort_constructors_first
+// #docregion class
+class Spacecraft {
```

Comment:
Is this supposed to extend `Describable`?

Reply:
Right now the `Describable` class exists on it's own. This isn't something I changed, though.

Reply:
It seems like it should, on second glance...

Reply:
Ah, but it would need to implement `describeWithEmphasis` as well. I vote to leave it as-is for now.

Reply:
Yeah either way is fine by me! I was just curious if that was the intention. 

I thought it wouldn't need to implement `describeWithEmphasis` if using `extend Describable`, as the class provides a default implementation. If you use `implements Describable` though that would then require an implementation.

Reply:
ah right. I don't actually know what the intention was here, but I suspect it should extend Describable